spec/controllers/settings_controller_spec.rb

```diff
@@ -1,32 +1,41 @@
-require 'spec_helper'
-
-describe SessionsController do
+ require 'spec_helper'
+
+describe SettingsController, type: :controller do
+   
+  let(:user) { FactoryGirl.create(:user) }
+=begin
+  let(:settings) do
+    {
+      :name => "name",
+      :email => "e@m.com"
+    }
+  end
+    
 
-  let(:user) { create(:user) }
+  let(:settings_params) { settings }
+=end
 
-  describe "#direct" do
-    it "redirects to dashboard_path if signed_in?" do
-      sign_in(user)
-      get :direct
-      expect(response).to redirect_to(dashboard_path)
-    end
+  before(:each) do
+    @settings = { :name => "name", :email => "e@m.com"},
```

Comment:
Done..

On Thu, Apr 9, 2015 at 1:18 PM, Ben Woodall <notifications@github.com>
wrote:

> In spec/controllers/settings_controller_spec.rb
> <https://github.com/benwoody/shrtnr/pull/4#discussion_r28098542>:
>
> >
> > -  describe "#direct" do
> > -    it "redirects to dashboard_path if signed_in?" do
> > -      sign_in(user)
> > -      get :direct
> > -      expect(response).to redirect_to(dashboard_path)
> > -    end
> > +  before(:each) do
> > +    @settings = { :name => "name", :email => "e@m.com"},
>
> It's best practice to define methods using the let syntax with RSpec.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benwoody/shrtnr/pull/4/files#r28098542>.
>



-- 
Joe Simpson
“Reasonable people adapt themselves to the world. Unreasonable people
attempt to adapt the world to themselves. All progress, therefore, depends
on unreasonable people.”
George Bernard Shaw